core/src/main/scala/cats/collections/BitSet.scala

```diff
@@ -117,6 +306,13 @@ sealed abstract class BitSet { lhs =>
         false
     }
 
+  /**
+   * Universal hash code.
+   *
+   * Bitsets that are the equal will hash to the same value. As in
+   * `equals`, the values present determine the hash code, as opposed
+   * to the tree structure.
+   */
   override def hashCode: Int = {
```

Comment:
Maybe this should be a lazy Val...

Reply:
I'm not 100% convinced, so let's wait on that.